geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/RepeatedRebalanceDUnitTest.java

```diff
@@ -0,0 +1,216 @@
+        + " --partition-resolver=" + PARTITION_RESOLVER
+        + " --colocated-with=" + COLOCATED_REGION_ONE)
+        .statusIsSuccess();
+  }
+
+  @After
+  public void after() {
+    for (int i = 0; i < memberList.size(); i++) {
+      memberList.get(i).stop();
+    }
+  }
+
+  @Test
+  public void testSecondRebalanceIsNotNecessaryWithAddedMembers() {
+
+    addDataToRegion(NUMBER_OF_ENTRIES);
+
+    addOrRestartServers(2, 0);
+
+    // Because we have 2 redundant copies and begin with 4 servers, redundancy is already satisfied
+    // before this rebalance. As such
+    // we expect to see no redundant copies created.
+    gfsh.executeAndAssertThat("rebalance").containsOutput(
+        "Total bytes in all redundant bucket copies created during this rebalance                        | 0")
+        .containsOutput(
+            "Total time (in milliseconds) spent creating redundant bucket copies during this rebalance       | 0")
+        .containsOutput(
+            "Total number of redundant copies created during this rebalance                                  | 0")
+        .doesNotContainOutput(
+            "Total bytes in buckets moved during this rebalance                                              | 0")
+        .doesNotContainOutput(
+            "Total time (in milliseconds) spent moving buckets during this rebalance                         | 0")
+        .doesNotContainOutput(
+            "Total number of buckets moved during this rebalance                                             | 0")
+        .doesNotContainOutput(
+            "Total time (in milliseconds) spent switching the primary state of buckets during this rebalance | 0")
+        .doesNotContainOutput(
+            "Total primaries transferred during this rebalance                                               | 0");
+    System.out.println("DEBR Finished rebalance one");
+
+    gfsh.executeAndAssertThat("rebalance").containsOutput(
+        "Total bytes in all redundant bucket copies created during this rebalance                        | 0")
+        .containsOutput(
+            "Total time (in milliseconds) spent creating redundant bucket copies during this rebalance       | 0")
+        .containsOutput(
+            "Total number of redundant copies created during this rebalance                                  | 0")
+        .containsOutput(
+            "Total bytes in buckets moved during this rebalance                                              | 0")
+        .containsOutput(
+            "Total time (in milliseconds) spent moving buckets during this rebalance                         | 0")
+        .containsOutput(
+            "Total number of buckets moved during this rebalance                                             | 0")
+        .containsOutput(
+            "Total time (in milliseconds) spent switching the primary state of buckets during this rebalance | 0")
```

Comment:
Are any of these strings extracted into variables anywhere?  The idea being, use them if possible that way if the command changes output, we wouldn't have to modify the tests individually.

Reply:
Turns out these do have CLI strings. I've added them to the assertions, although the exact formatting of the number isn't part of the extracted variables so we still have to hardcode that aspect.